packages/components/src/components/primitives/Link/Link.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Box, Link as ChakraLink, useStyleConfig } from '@chakra-ui/react';
+
+import { LinkProps } from './types';
+import { marginSizes, linkIconSizes } from './types';
+
+export const Link: FC<LinkProps> = ({
+  children,
+  icon,
+  iconPosition = 'left',
+  size = 'm',
+  ...props
+}) => {
+  const { textStyle } = useStyleConfig('Link', {
+    size: size,
+  });
```

Comment:
Suggestion: We are taking a different approach to implement `textStyles`, you can check that in `packages/components/src/theme/ComposerTheme/Components/Button.ts` starting in line 65.